internal/database/repos.go

```diff
@@ -457,6 +457,9 @@ type ReposListOptions struct {
 	// IDs of repos to list. When zero-valued, this is omitted from the predicate set.
 	IDs []api.RepoID
 
+	// Limits the set of results to repositories added by the user.
```

Comment:
go convention to start a docstring with the field name.
```suggestion
	// UserID limits the set of results to repositories added by the user.
```